### DISCUSSION TITLE
App store is not showing any app.

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#3879] [Questions] [answered]
### Steps to reproduce
1. Login as admin user
2. Click on the avatar icon (top right)
3. Click on Apps
4. Click on any category (except App bundles)

### Expected behavior 
Should be able to see apps available to download/update etc.

### Actual behavior 
Doesn't show any app.

### Host OS 
Ubuntu Server 22.04.3 LTS running in a VM on Proxmox 8.1.3

#### Nextcloud AIO version 
Nextcloud AIO v7.7.1

#### Current channel 
You are running the [latest](https://github.com/nextcloud/all-in-one#how-to-switch-the-channel) channel.

#### Other valuable info 
I wanted to install Nextcloud Notes app but the store seems to not show anything, I tried re-starting server/VM but no luck. Here is a screenshot of the store,
![2023-12-05_21-27](https://github.com/nextcloud/all-in-one/assets/53279454/7fbd40e7-a644-4431-8dd8-100f7e553e76)

**Comment by nfrawley93** [comment]
Having the same issue at the moment. When I select "App Bundles" it shows current ones, but any others will show empty pages similar to above. Only thing I see is when inspecting the page, any of the pages that show blank have this error.

I've tried accessing the App page with multiple accounts on my instance and they have the same issue.

`vue.runtime.esm.js:3049 TypeError: Cannot read properties of null (reading 'toLowerCase')
    at AppList.vue:135:1
    at Array.filter (<anonymous>)
    at o.apps (AppList.vue:135:1)
    at e.get (vue.runtime.esm.js:3446:33)
    at e.evaluate (vue.runtime.esm.js:3547:27)
    at o.apps (vue.runtime.esm.js:5537:25)
    at o.hasPendingUpdate (AppList.vue:128:1)
    at e.get (vue.runtime.esm.js:3446:33)
    at e.evaluate (vue.runtime.esm.js:3547:27)
    at o.hasPendingUpdate (vue.runtime.esm.js:5537:25)`

Edit: Leaving this here because I've seen a few posts regarding this over the years. I have changed nothing and it is now working as of an hour later. During this time my other Nextcloud instance never faced any issues so I am still unsure what to think.

**Comment by szaimen** [comment, marked as answer]
This problem occurs sometimes if the appstore is not reachable. This needs to be fixed either in nextcloud/server or in the appstore or in your netowrking infrastructure but not on AIO.

**Reply by apoorv569** [reply]
I see. Should I report this issue on nextcloud/server repo then?

**Reply by szaimen** [reply]
> I see. Should I report this issue on nextcloud/server repo then?

If you want but I think there are already multiple reports open regarding this.
